defaults/main.yaml

```diff
@@ -49,169 +49,194 @@ jenkins_pipeline_library_debug: false
 
 # Plugins and their versions that must be present for jenkins-pipeline-library
 jenkins_pipeline_library_plugins_present:
+  - name: analysis-model-api
```

Comment:
These are not alphabetically ordered. I know, "authentication-tokens" isn't either, but as we already touch it anyway...

Reply:
Solved